pkgs/racket-doc/scribblings/guide/hash-languages.scrbl

```diff
@@ -362,150 +362,77 @@ The @filepath{kiddo.rkt} example illustrates how the format for
 printing a result value can depend on the main module of a program
 instead of the language that is used to implement it.
 
+More broadly, certain features of a language are only invoked when
+a module written in that language is run directly with @exec{racket}
+(as opposed to being imported into another module). One example is
+result-printing style (as shown above). Another example is REPL
+behavior. These features are part of what's called the
+@emph{run-time configuration} of a language.
```

Comment:
In general you don't want to use `@emph` for technical terms. But I'm not sure if that's the right place to link to -- it seems like maybe this is the definition.